gtests/net/mptcp/add_addr/add_addr4_client.pkt

```diff
@@ -24,10 +24,10 @@
 // time, causing the host to retransmit its last data packet. Here we ensure
 // this previous packet is acked then we can send the ADD_ADDR
 +0.0      <   .  1:1(0)  ack 101  win 256  <nop, nop, TS val 705 ecr 305,                                    dss dack8=101 dll=0 nocs>
-+0.0      <   .  1:1(0)  ack 101  win 256  <nop, nop, TS val 705 ecr 305, add_address addr[saddr] hmac=auto>
++0.0      <   .  1:1(0)  ack 101  win 256  <nop, nop, TS val 705 ecr 305, add_address address_id=12 addr[ep=inet_addr("198.51.100.2")] hmac=auto>
```

Comment:
I think it's better than `addr[saddr]` because it assesses that the content of `ADD_ADDR` is exactly what we previously configure via netlink (otherwise, the value of `saddr` is "learned" by the live packet). There is no disadvantage of using a fixed address for `ADD_ADDR` content, unless we fear that a subsequent `MP_JOIN SYN` can potentially race with the `close()`. But this can also be fixed easily, either by changing the MPTCP limits in the client namespace, or by adding a blackhole (or unreachable?) route to the client for that address - in this way, we are 100% sure that that `SYN` will not be emitted.